### DISCUSSION TITLE
Welcome to OMSimulator Discussions!

### DISCUSSION THREAD

**Discussion by lochel** [OpenModelica/OMSimulator#894] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by superlou** [comment]
Hey, everyone! I've been using OMSimulator to build a physically-based simulation of a spaceship with a web front-end.

![Web client](https://i.imgur.com/Win1OKs.gif)

I'm exporting CS FMUs from OpenModelica and stepping them via the Python API. My background is electrical, so if anyone has tips on modeling fluids like an enclosed gas volume with slowly changing concentrations of oxygen and carbon dioxide, I'm all ears.
